app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -228,14 +226,17 @@ public void stopDiscovering() {
     }
 
     /**
-     * Called when a new bluetooth device is connected
+     * Called when the device is connected over Bluetooth
      *
-     * @param event Details about the event
+     * @param event Describes the event
      */
-    @Subscribe
+    @Subscribe(threadMode = ThreadMode.MAIN)
     public void onEvent(final BluetoothConnectedEvent event) {
```

Comment:
Already done in AndroidBluetoothService. If I log it in every class that listens to this event as well, we would have many duplicate logs.